server.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express'),
+    bodyParser = require('body-parser'),
+    fs = require('fs'),
+    app = express();
+    
+let stringifyFile = '';
```

Comment:
try to don't use global variables - using global variables isn't good idea, always ;)